pom.xml

```diff
@@ -837,5 +844,48 @@
         <defaultGoal>clean install</defaultGoal>
       </build>
     </profile>
+    <profile>
+      <id>jreleaser</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.jreleaser</groupId>
+            <artifactId>jreleaser-maven-plugin</artifactId>
+            <version>${jreleaser-maven-plugin.version}</version>
+            <inherited>false</inherited>
+            <configuration>
+              <projectVersionPattern>SEMVER</projectVersionPattern>
+              <jreleaser>
+                <release>
+                  <github>
+                    <owner>smallrye</owner>
+                    <name>smallrye-reactive-messaging</name>
+                    <overwrite>true</overwrite>
+                    <changelog>
+                      <preset>conventional-commits</preset>
```

Comment:
You might want to check the changelog output `jreleaser:changelog`). Another option is to adopt conventional commits going forward.

Reply:
Even without adopting conventional commits it seemed ok. But yes that's an option.